common/flatpak-dir.c

```diff
@@ -8290,8 +8368,12 @@ flatpak_dir_uninstall (FlatpakDir                 *self,
     {
       g_set_error (error, FLATPAK_ERROR, FLATPAK_ERROR_NOT_INSTALLED,
                    _("%s branch %s is not installed"), name, parts[3]);
+      return FALSE;
```

Comment:
Can you make this a separate commit so we can backport this to 1.0.x?

Reply:
I will